.ci/release/util.sh

```diff
@@ -3,14 +3,12 @@
 check_version() {
   v=${1:-}
 
-  if [[ "${v}" == "" ]]; then
-    echo "usage $0 <version>" # here $0 will be the calling script
-    echo "where <version> in format '1.2.3'"
+  if [ -z "${v}" ]; then
+    >&2 echo "The environment variable 'RELEASE_VERSION' isn't defined"
```

Comment:
what's the reason for `>&2` but it's not used in the `L10`?

Reply:
I forgot to do the same for the second message.
Ref: 3261b91cc1b0d6d6610d4106c26bd62a658e6579